src/components/MainNavBar/MainNavBar.js

```diff
@@ -35,7 +36,7 @@ function getMenuItem(MenuItem, activeSubmenuId, setSubMenuVisibility) {
               id={id}
               onFocus={() => setSubMenuVisibility(MenuItem.id, true)}
             >
-              {Icon && <Icon aria-hidden="true" />}
+              {Icon && <Icon fill={Colors.neutralGrey4} aria-hidden="true" />}
```

Comment:
In the design these icons are black. So is the text.

Reply:
Which design are you referencing? The text is indeed black but the icons not? https://projects.invisionapp.com/d/main#/console/16532745/351680895/inspect